.gitignore

```diff
@@ -8,3 +8,7 @@
 /htdocs/search/mysite/conf/
 /htdocs/search/sphinx/
 /vendor/
+
+.idea/
```

Comment:
@edlinklater thoughts here? should we tidy this before getting api deployed. Also... what is the default for api? I think it should be 3.1 until 3.2 goes stable, is that the case? @dhensby thoughts around this?